inclusive-design/wecount.inclusivedesign.ca
# ci: add GitHub action to ensure that the CMS branch is set properly

## Description

This PR adds a GitHub Action which monitors the Netlify CMS configuration file for changes and automatically adjusts the configuration to target the `main` branch for production and to target the `dev` branch for all other environments.

## Steps to test

1. Commit a change to the `src/admin/config.yml` to either `main` or `dev`.
2. Observe the results of the GitHub action.

**Expected behavior:** The CMS branch configuration should always be reset to `main` on `main` and `dev` on `dev`.

## Additional information

Uses [yq](https://mikefarah.gitbook.io/yq/) for parsing and updating the Netlify CMS configuration file.

## Related issues

Related to 54d638faa3f7aaadf1beeab87dd491819d4d56ab.
